platform/default/glfw_view.cpp

```diff
@@ -164,9 +165,12 @@ void GLFWView::onKey(GLFWwindow *window, int key, int /*scancode*/, int action,
         case GLFW_KEY_Q:
             view->clearAnnotations();
             break;
-        case GLFW_KEY_P: {
+        case GLFW_KEY_P:
             view->addRandomCustomPointAnnotations(1);
-        } break;
+            break;
+        case GLFW_KEY_L:
+            view->addRandomLineAnnotations(1);
```

Comment:
I believe that is meant to be _custom point_ because we're using a random, custom sprite image for the point symbols, whereas in this line annotation we use regular, non-custom line annotations.